Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM       node:16 as builder
-RUN        npm i -g --unsafe-perm @subql/node@0.23.1
+RUN        npm i @subql/node@0.31.1 --location=global
 
 FROM       node:16-alpine
 ARG        SORA_ENDPOINT
+ENV        NODE_VERSION=16.14.2
+ENV        YARN_VERSION=1.22.18
 ENV        TZ utc
-RUN        apk add --no-cache tini git
+RUN        apk add --no-cache tini git curl
```

Comment:
Why do you need curl?

Reply:
When I was looking for the reason why our image is not working. I asked for the composition of the official image of onfinality. They install curl there, not sure if I know why.

Reply:
Check their docker-compose. maybe for a healthcheck

Reply:
Yes, because in the npm image, npm-ci-login is installed, which is later launched. If you don't do this step, the build will fail.

Reply:
we can do healthchecks without curl, like here https://github.com/soramitsu/nbc-deploy/blob/develop/fastcore/docker-compose.yml

Reply:
> we can do healthchecks without curl, like here https://github.com/soramitsu/nbc-deploy/blob/develop/fastcore/docker-compose.yml

I understand correctly that it works if we have an endpoint?

Reply:
if there is no purpose, remove it

Reply:
and why do you need git?

Reply:
edited